build-farm/platform-specific-configurations/mac.sh

```diff
@@ -58,6 +58,8 @@ else
     if [ "${ARCHITECTURE}" == "x64" ]; then
       # We can only target 10.9 on intel macs
       export CONFIGURE_ARGS_FOR_ANY_PLATFORM="${CONFIGURE_ARGS_FOR_ANY_PLATFORM} --with-extra-cxxflags=-mmacosx-version-min=10.9"
+    elif [ "${ARCHITECTURE}" == "arm64" ]; then
```

Comment:
`arm64` and not `aarch64` correct? (just based on the appended command-line arg this modifies)

Reply:
arm64 is correct. It's what Apple refers to it as (along with all the other JVM vendors)